### ISSUE TITLE
“Debug Info” modal

### ISSUE THREAD

**Issue by outoftime** [popcodeorg/popcode#1604]
In many cases it would be helpful for students to quickly get debugging information when working with Popcode’s developers to try to resolve issues. So, let’s create a “debug info” feature, accessible from the hamburger menu, that pops up a modal with the following information (plus any more that we can think of):

* User ID (if logged in)
* Raw user agent
* Parsed user agent info from Bowser
* Viewport dimensions

A one-click copy button would be particularly useful here.

**Comment by laxmy** [comment]
Hi outoftime,
I am looking to contribute to Popcode and would like to give this item a go. Could you please assign this to me?

**Comment by outoftime** [comment]
@laxmy yep! just added you as a collaborator to the repo, can assign the issue once you’ve accepted the invite. thanks!
